.github/workflows/iOSBuild.yml

```diff
@@ -0,0 +1,40 @@
+name: iOS Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+
+concurrency:
+  group: ios-build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-ios:
+    runs-on: macos-13
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Select Xcode
+        run: sudo xcode-select -s '/Applications/Xcode_15.0.app'
+
+      - uses: ./.github/actions/setup-java
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@250fcd6a742febb1123a77a841497ccaa8b9e939 # v1.152.0
+        with:
+          bundler-cache: true
+
+      - name: Build KMM Module
+        working-directory: app-ios
+        run: bundle exec fastlane shared
+
+      - name: Build App
+        working-directory: app-ios
+        run: |
+          xcodebuild build -project App/DroidKaigi2023/DroidKaigi2023.xcodeproj \
```

Comment:
could you put these command to Fastlane ?

Reply:
done! https://github.com/DroidKaigi/conference-app-2023/pull/221/commits/ed91f46c89e8b60e2f763f2bf29730e76c48153c

Unfortunately, the build result of `xcodebuild` is so unstable.
So I decided to skip the build app. https://github.com/DroidKaigi/conference-app-2023/pull/221/commits/2c264a68ab252a91056736b6de1e019c77776e16